grails-app/utils/au/org/emii/portal/HtmlCharacterEntityReferences.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
```

Comment:
Furthermore, according to this document http://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses the Apache License 2.0 and the GNU GPL are compatible given the clause mentioned above